include/openthread.h

```diff
@@ -1802,7 +1673,7 @@ OTAPI void OTCALL otPlatformReset(otInstance *aInstance);
  *
  * @param[in]  aInstance A pointer to an OpenThread instance.
  */
-void otFactoryReset(otInstance *aInstance);
+OTAPI void OTCALL otFactoryReset(otInstance *aInstance);
```

Comment:
Can you please explain what you mean saying that openthread.h is the actual implementation?

All I'm asking here for is to keep a windows-specific code/syntax away from openthread's core implementation and API. Please note that other OS-specific solutions are not getting added directly into OT. Thus, having a separate wrapper just for windows sounds as a reasonable requirement, don't you think?